launch/ogc_gndtest.launch

```diff
@@ -0,0 +1,66 @@
+<launch>
+
+	<!-- Common OGC args -->
+	<arg name="log_output" default="screen" />
+	<arg name="interval_1" default="10"/>
+	<arg name="interval_2" default="60"/>
+	<arg name="interval_3" default="1"/>
+	<arg name="router_hostname" default="root@192.168.1.1" />
+	
+	<!-- sms args -->
+	<arg name="whitelist" default="$(find sms)/src/whitelist.txt"/>
+	<arg name="client_phone_no" default="'+12345678'"/>
+
+	<!-- tele args -->
+	<arg name="tdlib_auth_dir" default="/home/lenovo/.tdlib"/>
+	<arg name="monitored_chat" default="tele_link_grp" />
+	<arg name="command_whitelist" default="[Rumesh,Lau]" />
```

Comment:
Do we want to keep the names?

Reply:
Eventually the names will be removed when your code goes in; and I won't classify usernames as sensitive info, should be ok to keep it here